lumicks/pylake/detail/image.py

```diff
@@ -3,6 +3,60 @@
 import numpy as np
 
 
+class ImageMetaData:
+    """Image metadata
+
+    Parameters
+    ----------
+    pixel_size_x : float
+        Horizontal pixel size [nm]
+    pixel_size_y : float
+        Vertical pixel size [nm]
+        When omitted, assumed to be identical to pixel_size_x
+    pixel_time : float
+        How long does pixel acquisition take [ms]
+    """""
+    def __init__(self, pixel_size_x=1.0, pixel_size_y=None, pixel_time=1.0):
+        self._pixel_time = pixel_time
+        self._pixel_size_x = pixel_size_x
+        if pixel_size_y:
+            self._pixel_size_y = pixel_size_y
+        else:
+            self._pixel_size_y = pixel_size_x
+
+    @staticmethod
+    def from_dataset(json=None):
+        """
+        Fetch metadata from json structure
+
+        Parameters
+        ----------
+        json : json structure containing kymograph metadata
+        """
+        if json:
+            pixel_size = json["scan volume"]["scan axes"][0]["pixel size (nm)"]
+            pixel_time = json["scan volume"]["pixel time (ms)"]
+            return ImageMetaData(pixel_size_x=pixel_size, pixel_time=pixel_time)
+        else:
+            return ImageMetaData()
+
+    @property
+    def resolution(self):
+        """
+        :return: X, Y resolution in pixels per cm followed by unit specification accepted by Tifffile
```

Comment:
`:return:` is not part of `numpydoc`. For functions, you should use the `Returns` section. For properties, you can just say what it returns without any extra qualification. E.g. here it would be simply:
```python
@property
    def resolution(self):
        """X, Y resolution in pixels per cm followed by unit specification accepted by Tifffile"""
```

Reply:
Alright. Thanks. I'll keep that in mind.